templates/html/.partials/sidebar.html

```diff
@@ -25,18 +23,71 @@ <h1 class="text-2xl font-thin">{{asyncapi.info.title}} {{asyncapi.info.version}}
       </li>
       {{/if}}
     </ul>
-
     {{#if asyncapi.channels}}
     <h2 class="text-xs uppercase text-grey mt-10 mb-4 font-thin">Operations</h2>
     <ul class="text-sm list-reset mt-2">
+      {{#if params.sidebarOrganization}}
+      <!-- With tags in sidebar -->
+      {{#each asyncapi.tags as |categorytags|}}
+      <div class="{{#if open}}is-open{{/if}}">
```

Comment:
Would you mind adding a top margin to tag blocks?

```suggestion
      <div class="mt-4 {{#if open}}is-open{{/if}}">
```

Reply:
Sure, no problem. Makes it look cleaner.